src/routes/redux/detailsSlice.js

```diff
@@ -0,0 +1,68 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const detailsSlice = createSlice({
+  name: "details",
+  initialState: {
+    value: {
+      name: "",
+      lastName: "",
+      phone: "",
+      address: "",
+      shirtSize: "",
+      pantsSize: "",
+      budget: "",
+      neverWear: [],
+      alwaysWear: [],
+    },
+  },
+  reducers: {
```

Comment:
in terms of naming, i think it'll be better to name the reducers for their exact functionality - add*** or remove*** instead of update***
e.g. updateDetails is actually adding a single detail so i would rename to addDetail and updateNeverWearDelete -> removeNeverWearField or removeNeverWearEntry